### ISSUE TITLE
term: Accessibility

### ISSUE THREAD

**Issue by wion** [content-strategy-forum/csf-glossary#10]
I propose we change this term label to **Content accessibility** to be more specific about the type of accessibility we mean (e.g. content accessibility vs. wheelchair accessibility). This would also distinguish it a bit from the LoCS, thus we could consider our term a non-equivalent, exactly, and remove it from the overlapping terms list.

I would also propose using a limited number of examples in the definition (3 max) concerned with text, images, videos, and/or audio and _not_ any concerning presentational aspects alone (e.g. color contrast).

### References

In addition to the LoCS page as 1 reference (of the minimum 2 needed), another should be a source for web accessibility, and their are many standards bodies on this subject. Any other sources that focus on a given example used would be desirable too.
